[CI] EsqlSpecIT test {csv-spec:embedding.embedding with multiple knn queries in fork} failing

**Build Scans:**
- [elasticsearch-intake #37422 / part3](https://gradle-enterprise.elastic.co/s/nlxjsgnr42iia)
- [elasticsearch-intake #37423 / part3](https://gradle-enterprise.elastic.co/s/5bqspmtnvytbs)
- [elasticsearch-pull-request #136496 / part-3](https://gradle-enterprise.elastic.co/s/3vgja6srrvsgi)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecIT" -Dtests.method="test {csv-spec:embedding.embedding with multiple knn queries in fork}" -Dtests.seed=EA48E53B8FB47F6F -Dtests.locale=mt-Latn-MT -Dtests.timezone=Atlantic/Reykjavik -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Aembedding.embedding%20with%20multiple%20knn%20queries%20in%20fork%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Data mismatch:
row 0 column 0:a list containing
row 0 column 0:0: expected "live long and prosper" but was "be excellent to each other"
row 0 column 1:a list containing
row 0 column 1:0: expected "50.0" but was "45.0"
row 0 column 1:1: expected "57.0" but was "55.0"
row 0 column 1:2: expected "56.0" but was "54.0"
row 0 column 2:a list containing
row 0 column 2:0: expected "fork2" but was "fork1"
row 1 column 0:a list containing
row 1 column 0:0: expected "be excellent to each other" but was "live long and prosper"
row 1 column 1:a list containing
row 1 column 1:0: expected "45.0" but was "50.0"
row 1 column 1:1: expected "55.0" but was "57.0"
row 1 column 1:2: expected "54.0" but was "56.0"
row 1 column 2:a list containing
row 1 column 2:0: expected "fork1" but was "fork2"
Actual:
text_field:text                                                       | query_embedding:dense_vector | _fork:keyword
be excellent to each other                                     
[truncated]
```

**Issue Reasons:**
- [main] 3 consecutive failures in test test {csv-spec:embedding.embedding with multiple knn queries in fork}
- [main] 2 consecutive failures in step part3
- [main] 1 consecutive failures in step part-3
- [main] 2 consecutive failures in pipeline elasticsearch-intake
- [main] 1 consecutive failures in pipeline elasticsearch-pull-request
- [main] 3 failures in test test {csv-spec:embedding.embedding with multiple knn queries in fork} (100.0% fail rate in 3 executions)
- [main] 2 failures in step part3 (100.0% fail rate in 2 executions)
- [main] 2 failures in pipeline elasticsearch-intake (100.0% fail rate in 2 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -232,15 +232,6 @@ tests:
 - class: org.elasticsearch.xpack.entsearch.EnterpriseSearchRestIT
   method: test {p0=entsearch/rules/40_rule_query_search/Perform a rule query with an organic query that must be rewritten to another query type}
   issue: https://github.com/elastic/elasticsearch/issues/146106
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecIT
-  method: test {csv-spec:embedding.embedding with multiple knn queries in fork}
-  issue: https://github.com/elastic/elasticsearch/issues/146130
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecIT
-  method: test {csv-spec:embedding.embedding with multiple knn queries in fork}
-  issue: https://github.com/elastic/elasticsearch/issues/146130
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:embedding.embedding with multiple knn queries in fork}
-  issue: https://github.com/elastic/elasticsearch/issues/146137
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=tsdb/25_id_generation/*}
   issue: https://github.com/elastic/elasticsearch/issues/146177

